### PR TITLE
22.8 review controller fix

### PR DESCRIPTION
**Pull request by HiddenBrilliance**

### CODE REVIEW

**Review by ErinKace — APPROVED** [review]
Once these changes are combined with the other branch, this should work and display reviews!

**Review by e-karr — APPROVED** [review]
I was able to save a review, see it in the database, and view it on my profile view!

I'm sure there's a way to save more than one review. You might check out Erin's code for saving movies since she's able to save more than one. but you have the core functionality

I had to do some more merge conflicts when reviewing your branch that I didn't see in GitHub. might have just been my local machine. but you might look out for that before you merge. they were simple conflicts to resolve though

File: angular-ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -7,7 +7,11 @@ import { ProfileComponent } from './profile/profile.component';
 import { UserLoginComponent } from './user-login/user-login.component';
 import { UserRegisterComponent } from './user-register/user-register.component';
 import { MyListComponent } from './my-list/my-list.component';
+<<<<<<< HEAD
```
Comment (e-karr) [inline angular-ui/src/app/app-routing.module.ts:10]: not sure why the merge conflict syntax stayed in here after you resolved conflicts. but you'll want to keep line 13 here and can delete the >>>>>> and ===== parts

File: angular-ui/src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -44,8 +44,13 @@
             type="button"
             data-bs-toggle="dropdown"
             aria-expanded="false"
+<<<<<<< HEAD
```
Comment (e-karr) [inline angular-ui/src/app/navbar/navbar.component.html:47]: same here. no idea how this happened. but keep lines 51-52. don't need 47-50 and 53.

File: angular-ui/src/app/navbar/navbar.component.ts [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.css'],
 })
 
-//get the logged in variable data into here somehow .. 
 export class NavbarComponent implements OnInit {
   isLoggedIn: boolean;
+  currentUser: User | null;
 
+<<<<<<< HEAD
```
Comment (e-karr) [inline angular-ui/src/app/navbar/navbar.component.ts:15]: I think this is the last one that is strange. but keep lines 20-23. delete 15-19 and 24.

**Pull request merged**